nservicebus/operations/metrics-service-control.md

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
Seems that new page is not included in the menu, is that intentional?
If not, please add a new entry in menu.yml.